create/how_to/aws.md

```diff
@@ -0,0 +1,181 @@
+
+# Deploy a MeiliSearch Instance on AWS
+
+Using our **MeiliSearch [AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)**, MeiliSearch can be deployed on AWS in just a few minutes.
+The following `how-to guide` will walk you through every step to deploy MeiliSearch in an AWS EC2 instance.
+
+**Table of Contents**:
+
+[[toc]]
+
+## Part 1: Deploy an out-of-the-box MeiliSearch instance
+
+### 1. Launch an instance from the AWS console
+
+After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the **Compute** service. Then go to **EC2**, and finally open your **Instances** console.
+
+![Launch an instance from AWS console](/aws/01.launch-instances.png)
```

Comment:
The image quality isn't good. Not sure if it's the quality or the size. If it's the size that make it hard to read, we should link to the full size image. It's like this for all images.

Reply:
Are you sure it's a problem with the image itself? On my browser the deploy previews open very zoomed-out. A couple `⌘` + `+`s solves the problem.

Reply:
So as we discovered, it's the size of the image that is causing problem, so I did another PR to add a zoom features. It doesn't change the fact that it's blurry, but it give the opportunity for people to see the full resolution version!